app/components/work_packages/activities_tab/journals/item_component/details.rb

```diff
@@ -0,0 +1,208 @@
+          end
+        end
+
+        def render_header_end(header_container)
+          header_container.with_column(flex_layout: true) do |header_end_container|
+            render_notification_bubble(header_end_container) if has_unread_notifications
+            render_activity_link(header_end_container)
+          end
+        end
+
+        def render_notification_bubble(container)
+          container.with_column(mr: 2, classes: "bubble-container") do
+            bubble_html
+          end
+        end
+
+        def render_activity_link(container)
+          container.with_column(pr: 3) do
+            render(Primer::Beta::Link.new(
+                     href: activity_anchor,
+                     scheme: :secondary,
+                     underline: false,
+                     font_size: :small,
+                     data: { turbo: false }
+                   )) do
+              "##{journal.version}"
+            end
+          end
+        end
+
+        def icon_aria_label
+          journal.initial? ? "Add" : "Change"
+        end
+
+        def render_details(details_container)
+          return if skip_rendering_details?
+
+          details_container.with_row(flex_layout: true, pt: 1, pb: 3) do |details_container_inner|
+            if journal.initial?
+              render_empty_line(details_container_inner)
+            else
+              render_journal_details(details_container_inner)
+            end
+          end
+        end
+
+        def skip_rendering_details?
+          journal.initial? && journal_sorting == "desc"
+        end
+
+        def render_journal_details(details_container_inner)
+          journal.details.each do |detail|
+            render_single_detail(details_container_inner, detail)
+          end
+        end
+
+        def render_single_detail(container, detail)
+          container.with_row(flex_layout: true, my: 1, align_items: :flex_start) do |detail_container|
+            render_stem_line(detail_container)
+            render_detail_description(detail_container, detail)
+          end
+        end
+
+        def render_stem_line(container)
+          container.with_column(classes: "journal-detail-stem-line")
+        end
+
+        def render_detail_description(container, detail)
+          container.with_column(pl: 1, font_size: :small) do
+            render(Primer::Beta::Text.new(classes: "journal-detail-description")) { journal.render_detail(detail) }
+          end
+        end
+
+        def render_empty_line(details_container)
+          details_container.with_row(my: 1, font_size: :small, classes: "empty-line")
+        end
+
+        def bubble_html
+          "
+          <span
+            class=\"comments-number--bubble op-bubble op-bubble_mini\"
+            data-test-selector=\"user-activity-bubble\"
+          ></span>
+          ".html_safe # rubocop:disable Rails/OutputSafety
+        end
```

Comment:
Same as above, also extract this into a separate component if you use it multiple times

Reply:
Done